executor/src/witgen/jit/compiler.rs

```diff
@@ -77,8 +77,9 @@ pub fn compile_effects<T: FieldElement>(
 
     record_start("JIT-compilation");
     let start = std::time::Instant::now();
-    log::trace!("Calling cargo...");
-    let r = powdr_jit_compiler::call_cargo(&code);
+    let opt_level = 0;
```

Comment:
So now we can no longer measure the runtime speed of optimized binary?

Reply:
Do you do that regularly? If you want to, you can change this number here ;)

The thing is that `cargo nextest run` should just work without the need for a complicated setup where we tune environment variables.

Reply:
OK, fine with me now. I think it should be a CLI argument, but we can do that later. In the tests we'll always want to use 0 as long as we don't share compilation between tests.